src/apps/packet_filter/README.md.src

```diff
@@ -54,15 +74,11 @@ is allowed.
 
 — Key **state_track**
 
-*Optional*. Tracks connections state in a named connection table.  If used
-inside a rule, every packet that passes this specific rule is tracked.
-If used 'outside' any rule, packets passing _any_ rule on this app will be
-tracked.
+*Optional*. A string naming a connection table. If set, packets passing
+the rule are tracked in the specified connection table.
 
 — Key **state_check**
 
-*Optional*. Checks if a packet belongs to a connection in a named
-connection table.  If used inside a rule, a packet must belong to an
-existing connection in addition to any other condition in the rule to pass.
-If used 'outside' any rule, if a packet belongs to an existing connection
-it's enough to pass the app.
+*Optional*. A string denoting a connection table. If set, packets must
```

Comment:
This describes the following portion:

> If used inside a rule, a packet must belong to an
> existing connection in addition to any other condition in the rule to pass.

The "out of rule" behavior is described in the global `state_check` option.